### PR TITLE
Completed fetch-in-react exercise. -Lindsey Laveirge

### PR DESCRIPTION
**Pull request by llaveirge**
fetch-in-react

### CODE REVIEW

File: fetch-in-react/src/app.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users', { method: 'GET' })
+      .then(response => response.json())
+      .then(users => {
+        this.setState({ users: users });
+        this.setState({ isLoading: false });
```
Comment (scbowler) [inline fetch-in-react/src/app.jsx]: Instead of calling `setState` twice, it is better to call it once and update both values at the same time.

**Review by scbowler — CHANGES_REQUESTED** [review]
On the right track, but one update needed. Let me know if you have any questions.

File: fetch-in-react/src/app.jsx (hunk 1 above)
Comment (llaveirge) [reply fetch-in-react/src/app.jsx]: Right! Thanks!

**Comment by llaveirge** [comment]
Still works! 
image

**Review by scbowler — APPROVED** [review]
Much better! Looks good! 👍

**Pull request merged**